RecoBTag/Combined/python/caloDeepCSVJetTags_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+caloDeepCSVJetTags = cms.EDProducer(
+	'DeepFlavourJetTagsProducer',
+	src = cms.InputTag('caloDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
+	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json'),
+  toAdd = cms.PSet(
+      probcc = cms.string('probc')
+      ),
+	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(caloDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
```

Comment:
if you want to keep it in sync, please clone from the pf-based with an appropriate edit of the ```src``` value